src/components/form/input.js

```diff
@@ -61,6 +61,9 @@ const StyledError = styled.img`
         right: 2rem;
         top: 1.6rem;
     }
+    @media ${device.desktopL} {
+        top: 1rem;
+    }
```

Comment:
This will have a global affect across the whole site. Have this been verified across all other components that use this? @kevinw-binary

Reply:
as far as I checked it's fine, and we can ask QA to verify this